src/components/blocks/software/styles.css

```diff
@@ -1,15 +1,16 @@
 @import '@ably/ui/core/styles.css';
 @layer components {
   .docs-label {
-    @apply absolute bg-charcoal-grey p-4 ui-text-menu3;
+    @apply bg-charcoal-grey p-4 ui-text-menu3;
     color: #d9d9da;
   }
 
   .docs-language-label {
-    @apply top-4 right-4 docs-label cursor-default;
+    @apply absolute top-4 right-4 docs-label cursor-default;
```

Comment:
This `absolute` feels a little weird to me. Without the component also specifying the relative element, I think this is a bit error prone.

Reply:
This is part of the process of removing the `absolute` from another, more general class, we don't use this class any more so we can simply remove it from the code base.